packages/terra-button/tests/jest/Button.test.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
+import { clear, mockUserAgent } from 'jest-useragent-mock';
```

Comment:
I would rather not pull in another dependency for this, especially when it's something easily achievable on our end.

We can simply create a spy for navigator e.g.
https://stackoverflow.com/questions/52868727/how-to-mock-window-navigator-language-using-jest

Some more tips:
- Use `beforeAll` and `afterAll` blocks to set and [restore](https://jestjs.io/docs/mock-function-api#mockfnmockrestore) the spy.
- Use the `mockReturnValueOnce()` method to set the appropriate value for the spy.

Reply:
Made the change. Thank you!